yellowbrick/features/manifold.py

```diff
@@ -631,12 +637,17 @@ def manifold_embedding(
         alpha=alpha,
         random_state=random_state,
         colorbar=colorbar,
+        show=show,
```

Comment:
`show` is the parameter for the quick method and we do not have any show parameter in the original visualizer(`Manifold`). So this might not go here!